src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Stepper/StepperHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Xamarin.Platform.Handlers
+{
+	public partial class StepperHandler
+	{
+		public static PropertyMapper<IStepper> StepperMapper = new PropertyMapper<IStepper>(ViewHandler.ViewMapper)
```

Comment:
wouldn't it make more sense to define the mapper in StepperHandler.*.cs, that is closer to the implementation. This way we could also _not_ map properties that do not make sense in some platform, and thus have a capability (by code) to figure out if a property is mapped on a platform or not ?

Reply:
But we repeat the same mapping N times, many times being the same, right?